LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2013-2016 simplabs GmbH/Marco Otte-Witte and contributors
+Copyright (c) 2016-2017 simplabs GmbH and contributors
```

Comment:
well, looking at https://github.com/simplabs/ember-cli-pixijs/graphs/contributors that question essentially answers itself 😉